src/pages/game/index.tsx

```diff
@@ -0,0 +1,45 @@
+import styles from "./index.module.scss";
+import LayoutComponent from "src/components/global/layout.module";
+import { Game } from "@structs/game";
+import { prisma } from "src/server/prisma";
+import Link from "next/link";
+
+const Games = ({ games }: Props) => {
```

Comment:
Might be a good idea to introduce a consistent component / page definition pattern.

Next.js has a type for pages (`NextPage`), and React has the `React.FC` type for function components.

By doing this we ensure that components always return proper JSX.

```tsx
{/* A page (generic props type is optional) */}
const Games: NextPage<Props> = ({ games }) => {
  // ...
};


{/* Simple component without props (The React namespace doesn't have to be imported) */}
export const Information: React.FC = () => {
  // ...
};

{/* Simple component (generic props type is optional) */}
export const Button: React.FC<ButtonProps> = (props) => {
  // ...
};

{/* Simple component with children and props */}
import type { PropsWithChildren } from "react";

export const Card: React.FC<PropsWithChildren<CardProps>> = ({ children, ...props }) => {
  // ...
};
```